config/locales/oc.yml

```diff
@@ -206,8 +206,62 @@ oc:
   authorize_follow:
     error: O planhèm, i a agut una error al moment de cercar lo compte
     follow: Sègre
-    prompt_html: 'Avètz (<strong>%{self}</strong>) demandat de sègre :'
```

Comment:
`<strong>%{self}</strong>` is an important thing that shows user information. Do not want to delete it as much as possible